README.rst

```diff
@@ -271,6 +271,15 @@ has the css class ``inneredit`` and a data attribute named ``uid`` containing th
 
 After editing the content, the view automatically reloads the block.
 
+Additional plone.restapi methods
+--------------------------------
+
+After creating blocks in a simplelayout content page they have to be synchronized with the pages config. Otherwise
+they wont show up in the FrontEnd. It also removes objects which are in the pages config but not in the page itself.
+
+To do this, you can simply send a POST request to the path of your page, appended with
```

Comment:
You could also refer to the plone.restapi, as it is a standardized restapi endpoint.